### PR TITLE
New package: regina-rexx-3.9.1

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

**Comment by xtraeme** [comment]
- This needs a development subpkg
- The shared library soname must be added to `common/shlibs`

File: srcpkgs/regina-rexx/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+license="GPL-2"
+homepage="http://regina-rexx.sourceforge.net/"
+distfiles="${SOURCEFORGE_SITE}/project/regina-rexx/regina-rexx/3.9.1/Regina-REXX-3.9.1.tar.gz"
+checksum=5d13df26987e27f25e7779a2efa87a5775213beeda449a9efac59b57a5d5f3ee
+
+regina-rexx-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/lib/*.so
```
Comment (xtraeme) [inline srcpkgs/regina-rexx/template]: Development pkg must contain headers, static libraries, pkg-config files, etc. i.e files that are only required when compiling against the software, not at runtime.

Comment (diogoleal) [inline srcpkgs/regina-rexx/template]: Sorry. 
I made the requested changed.

**Comment by pullmoll** [comment]
It looks like `common/shlibs` has changed. Please rebase and again add
`libregina.so.3 regina-rexx-3.9.1_1`
to `common/shlibs`, preferably without the empty line :smile:

**Pull request closed without merging**